tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/BulkheadSynchRetryTest.java

```diff
@@ -237,11 +241,26 @@ public void testBulkheadRetryClassSynchronous55() {
     public void testNoRetriesBulkhead() {
         int threads = 30;
         int maxSimultaneousWorkers = 5;
-        int expectedTasks = 10;
+        int expectedTasks = 5;
```

Comment:
Please explain why you changed this number?

Reply:
I changed this number because it's wrong. As I explained in my second comment, the number of 10 is **only** correct if it's working under asynchronous (thread pool style) bulkhead.

Having removed the `@Asynchronous` annotation, this test should only expect 5 threads to return: It fires off 30 threads to a bulkhead with a value of 5, so 5 should execute, and 25 should immediately return with bulkhead errors. Given that there's no retry on this method (_maxRetries=0_), none of those 25 errors should try again.

Previously, the method was annotated with a bulkhead of 5, a waiting task queue of 5, and the `@Asynchronous` annotation, so the previous number of 10 was the correct number to pass the test, but the test itself was wrong - it shouldn't be testing asynchronous behaviour. It would fire off 30 threads, 5 would execute, 5 more would wait, and 20 would return bulkhead errors.

As requested, I put in an extra test to check that without the `@Asynchronous` annotation the _waitingTaskQueue_ parameter was ignored. The git diff though shows it a little funny, so I'll try to explain:

1. I removed the `@Asynchronous` annotation and _waitingTaskQueue_ parameter from the original class to fix the test and remove the spurious parameter respectively - `BulkheadRapidRetry550MethodSynchBean.java`
2. I then changed the name of this class to match the naming convention already being used, from _550_ to _50_, as it no longer had 3 values (_value_, _waitingTaskQueue_, and _maxRetries_) - it's now called `BulkheadRapidRetry50MethodSynchBean.java` because it only has 2 values (_value=5_ and _maxRetries=0_)
3. I added in my new test which is almost identical to the original, just with `@Asynchronous` removed, so as to test that _waitingTaskQueue_ is ignored when the `@Asynchronous` annotation isn't present - it should behave exactly the same as the original test (now `BulkheadRapidRetry50MethodSynchBean.java`), returning 5.

Reply:
I think the changes above are good.